src/components/pages/metadata-view-page/components/MDVGridView.js

```diff
@@ -0,0 +1,122 @@
+// @flow
+import * as React from 'react';
```

Comment:
We may not need this and its parent, something directly can be used from https://bvaughn.github.io/react-virtualized/#/components/Collection perhaps. Will have to try it out. If we are using Grid or any other view as an abstraction, that should be an isolated PR and likely built within the /features folder.